pages/04_multitenancy_and_rls.md

```diff
@@ -0,0 +1,68 @@
+# Multi-tenancy and Row-level Security
+
+Scaling an embedded analytics deployment of Tableau often means providing content to different groups of users. There is a security implication of making sure the different groups do not see each others' data (and metadata). There is also a deployment and maintenance consideration: Common dashboards should have to be built only once, even if the different groups of users access different data with those common dashboards.
+
+### Multi-tenancy and Sites
```

Comment:
Added bulleted points on why you would choose one or the other. Also emailed a variety of people for an opinion on that guidance.